src/linear_algebra/quotient.lean

```diff
@@ -206,6 +206,9 @@ def mkq : M →ₗ[R] M ⧸ p :=
 
 @[simp] theorem mkq_apply (x : M) : p.mkq x = quotient.mk x := rfl
 
+@[simp] theorem mkq_eq_zero : p.mkq x = 0 ↔ x ∈ p :=
```

Comment:
Won't the linter reject this due to the lemma immediately above?

Reply:
I didn't realized CI hasn't finished yet. Let's see.